site/layouts/partials/home/masthead-followup.html

```diff
@@ -1,37 +1,39 @@
-<div class="container masthead-followup px-4 px-md-3">
-  <section class="row mb-5 pb-md-4 align-items-center">
-    <div class="col-md-5">
-      <div class="masthead-followup-icon d-inline-block mb-2 text-white bg-success">
-        {{ partial "icons/code.svg" (dict "width" "32" "height" "32") }}
-      </div>
-      <h2 class="display-5 fw-normal">Installation</h2>
-      <p class="lead fw-normal">
-        Install Bootstrap’s source Sass and JavaScript files via npm, Composer, or Meteor.
-      </p>
-      <p>Package managed installs don’t include documentation or our full build scripts. You can also <a href="https://github.com/twbs/bootstrap-npm-starter">use our npm template repo</a> to quickly generate a Bootstrap project via npm.</p>
-      <a class="btn btn-lg btn-outline-primary mb-3" href="/docs/{{ .Site.Params.docs_version }}/getting-started/download/">Read installation docs</a>
-    </div>
-    <div class="col-md-7 ps-md-5">
-      {{ highlight "npm install bootstrap" "sh" "" }}
-      {{ highlight (printf ("gem install bootstrap -v %s") .Site.Params.current_ruby_version) "sh" "" }}
+<div class="container masthead-followup">
+  <div class="col-lg-7 mx-auto pb-3 mb-3 mb-md-5 text-md-center">
+    <div class="masthead-followup-icon d-inline-block mb-3" style="color: var(--bs-indigo);">
+      <svg class="bi fs-1"><use xlink:href="#code"></use></svg>
     </div>
-  </section>
+    <h2 class="display-5 mb-3 fw-semibold lh-sm">Get started anyway you&nbsp;want</h2>
```

Comment:
```suggestion
    <h2 class="display-5 mb-3 fw-semibold lh-sm">Get started any way you&nbsp;want</h2>
```
any way (in any of the ways) vs anyway (regardless/nonetheless)